requestor/tests/test_message_verification.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
+
+from eth_account.messages import encode_defunct
+from eth_account import Account
+from requestor.server.app import validate_massage
```

Comment:
Mypy is not happy about this import. You have two options:
1) Change it to `from ..server.app` and add `__init__.py` in both `requestor` and `tests` directories. (That's kinda dirty)
2) Modify `.github/workflows/linters.yml` and add project top directory to `PYTHONPATH` when running `mypy` and `pylint`. (That's preferred)